app/src/main/java/com/github/se/cyrcle/ui/report/AdminScreen.kt

```diff
@@ -81,6 +98,89 @@ fun FilterSection(
       }
 }
 
+/**
+ * Displays details of a reported object based on its type: REVIEW, PARKING, or IMAGE.
+ *
+ * @param objectType The type of the reported object (REVIEW, PARKING, or IMAGE).
+ * @param objectUID The unique identifier of the reported object.
+ * @param userUID The unique identifier of the user associated with the report.
+ * @param reviewViewModel The ViewModel used to fetch review-related data.
+ * @param parkingViewModel The ViewModel used to fetch parking or image data.
+ */
+@Composable
+fun ReportDetailsContent(
+    objectType: ReportedObjectType,
+    objectUID: String?,
+    userUID: String?,
+    reviewViewModel: ReviewViewModel,
+    parkingViewModel: ParkingViewModel
+) {
+  when (objectType) {
+    ReportedObjectType.REVIEW -> {
+      var review by remember { mutableStateOf<Review?>(null) }
+      LaunchedEffect(objectUID) {
+        reviewViewModel.getReviewById(objectUID!!, { result -> review = result }, {})
```

Comment:
Is there no safe handling for `NullPointerExceptions`?

Reply:
But then why did you declare it's type `String?` if it cannot be null?

Reply:
Honestly, not sure anymore. I removed that in a latest version